### ISSUE TITLE
clearLine not found

### ISSUE THREAD

**Issue by mkuendig** [DeviaVir/zenbot#170]
I use the docker image on Kubernetes and I get this error:

2017-05-21T10:25:04.081962657Z fetching pre-roll data:
2017-05-21T10:25:04.922070694Z 
2017-05-21T10:25:04.922102346Z skipping 96 hrs of previously collected data
2017-05-21T10:25:04.923466585Z 
2017-05-21T10:25:04.92347838Z poloniex.ETH-USDT saved 200 trades 0 days left
2017-05-21T10:25:04.923678472Z 
2017-05-21T10:25:04.923687336Z download complete!
2017-05-21T10:25:04.92369051Z 
2017-05-21T10:25:04.930738242Z                DATE            PRICE     DIFF            VOL         RSI                         ACTIONS                      BAL                PROFIT
2017-05-21T10:25:05.024836115Z /app/lib/engine.js:542
2017-05-21T10:25:05.02488366Z       process.stdout.clearLine()
2017-05-21T10:25:05.024887219Z                      ^
2017-05-21T10:25:05.024890524Z 
2017-05-21T10:25:05.024893697Z TypeError: process.stdout.clearLine is not a function
2017-05-21T10:25:05.024896615Z     at writeReport (/app/lib/engine.js:542:22)
2017-05-21T10:25:05.02489929Z     at /app/lib/engine.js:673:17
2017-05-21T10:25:05.024902243Z     at Object.onPeriod (/app/extensions/trend_ema/strategy.js:74:7)
2017-05-21T10:25:05.024905049Z     at Array.<anonymous> (/app/lib/engine.js:662:35)
2017-05-21T10:25:05.024908719Z     at Immediate.each (/app/node_modules/run-series/index.js:17:24)
2017-05-21T10:25:05.0249113Z     at runCallback (timers.js:672:20)
2017-05-21T10:25:05.024914365Z     at tryOnImmediate (timers.js:645:5)
2017-05-21T10:25:05.024916845Z     at processImmediate [as _immediateCallback] (timers.js:617:5)

It works when I change the code in engine.js to:

```
      //process.stdout.clearLine()
       var readline = require('readline');
      readline.cursorTo(process.stdout, 0);
      //process.stdout.cursorTo(0)
     
```

**Comment by carlos8f** [comment]
ahh, ok. what version of node is this?

**Comment by mkuendig** [comment]
NODE_VERSION=7.10.0

**Comment by carlos8f** [comment]
fixed with 77fd503d90605afe05156127776d6308cc550ac7
